Makefile

```diff
@@ -153,6 +153,10 @@ lint: ## Run the linter on the codebase
 
 check: check_fmt lint test ## Check that the code conforms to all requirements for commit. Formatting, licenses, vet, tests and linters
 
+ready: fmt fmt_license lint test	
```

Comment:
wouldnt it make more sense to add `fmt_license` and `lint` to the `test` target?

Reply:
or remove `test` from this and run `ready` from `test`

Reply:
My original vision was to use `check` goal both in the CI and locally so that we have some common tooling for checking the readiness of the codebase. That's why `check` uses `check_fmt` and not `fmt` - it should fail if the code doesn't conform.

I meant `ready` as an alternative of the `check` goal for everyday work on the code. You don't want to merely check that formatting is OK, you want the formatting to be applied straight away.